packages/smart-contracts/test/lib/artifact.ts

```diff
@@ -0,0 +1,49 @@
+import { BigNumber, providers } from 'ethers';
```

Comment:
it won't be necessary to have more tests, this ensures the utilities are working and the types are correct